app/services/lifetime_usages/usage_thresholds/completion_rate.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module LifetimeUsages
+  module UsageThresholds
+    class CompletionRate < BaseService
+      def initialize(lifetime_usage:, usage_threshold:, largest_non_recurring_threshold_amount:)
```

Comment:
I guess this one is not really used :smile:

Reply:
🙈 Forgot to remove that one!